src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -72,19 +71,22 @@ class PhoneEditModal extends React.Component {
     let defaultFieldValue;
 
     if (this.props.data) {
-      defaultFieldValue = merge(this.props.data, {
-        inputPhoneNumber:
-          this.props.data &&
-          [this.props.data.areaCode, this.props.data.phoneNumber].join(''),
+      defaultFieldValue = {
```

Comment:
No need to use lodash `merge` here as the data prop will be a shallow object so I converted this to use an object spread and ES6 template strings instead of the clever Array.join hack that was being used.